add adapter urdex finance

Urdex-finance opened the pull request:
� Running URDEX adapter �
_______________________________________
Dexs for 7/6/2023
_______________________________________

ARBITRUM �
Backfill start time: 6/6/2023
NO METHODOLOGY SPECIFIED
Total volume: 285631.16928795597
Total fees: 1859.73562859
Daily volume: undefined
Daily fees: undefined
Timestamp: 1686096000

llamatester commented:
The urdex adapter exports: 
        
 
 _______________________________________
    Dexs for 7/6/2023
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 6/6/2023
    NO METHODOLOGY SPECIFIED
    Total volume: 285631.16928795597
    Total fees: 1860.23472479
    Daily volume: undefined
    Daily fees: undefined
    Timestamp: 1686096000
    
    
    
    
    Done in 6.11s.

dtmkeng commented:
hi ty for pr, could you check why daily valume and fees is `undefined`
if that day does not have value should set 0

llamatester commented:
The urdex adapter exports: 
        
 
 _______________________________________
    Dexs for 8/6/2023
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 6/6/2023
    NO METHODOLOGY SPECIFIED
    Total volume: 321681.77047925466
    Daily volume: 0
    Timestamp: 1686182400
    
    
    
    
    Done in 4.73s.

Urdex-finance commented:
> 

Hi, ty for your comment, i updated my pr,
i set default volume is 0 when got no data from our server,
i removed fee because of some reason from our side.
Thanks,

Urdex-finance commented:
> is it derivative volume or swap volume?

This is derivative volume

Urdex-finance commented:
> > > is it derivative volume or swap volume?
> > 
> > 
> > This is derivative volume
> 
> will list on this https://defillama.com/derivatives but this menu does not show on defillana page now

TY for your information,
Our volume will be shown in this page after this pr is merged, is it correct ?

Urdex-finance commented:
> > > > > is it derivative volume or swap volume?
> > > > 
> > > > 
> > > > This is derivative volume
> > > 
> > > 
> > > will list on this https://defillama.com/derivatives but this menu does not show on defillana page now
> > 
> > 
> > TY for your information, Our volume will be shown in this page after this pr is merged, is it correct ?
> 
> yeah, just test some date value is 0 which date have dailyVolume ?

dailyVolume available from 1686268800

llamatester commented:
The urdex adapter exports: 
        
 
 _______________________________________
    Dexs for 10/6/2023
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 6/6/2023
    NO METHODOLOGY SPECIFIED
    Total volume: 640550.6766892297
    Daily volume: 64072.08927904031
    Timestamp: 1686355200
    
    
    
    
    Done in 4.30s.
